src/common/relayd/relayd.c

```diff
@@ -117,9 +117,63 @@ static int recv_reply(struct lttcomm_relayd_sock *rsock, void *data, size_t size
 error:
 	return ret;
 }
+/*
```

Comment:
Add an empty line between this comment block and the previous function.

Reply:
done